cassandra/src/test/java/org/apache/calcite/test/CassandraAdapterTest.java

```diff
@@ -84,8 +86,8 @@
   private static boolean enabled() {
     final boolean enabled = CalciteSystemProperty.TEST_CASSANDRA.value();
     Bug.upgrade("remove JDK version check once current adapter supports Cassandra 4.x");
-    final boolean compatibleJdk = TestUtil.getJavaMajorVersion() != 11
-                                      && TestUtil.getJavaMajorVersion() != 12;
+    final List<Integer> incompatibleJdks = Arrays.asList(11, 12, 13);
```

Comment:
Maybe it's easier to do:
```java
 final boolean compatibleJdk = TestUtil.getJavaMajorVersion() < 11;
```

Reply:
yea so I had thought about that. I think JDK 8 comes back as 1.8 sometimes? I'd need to double check. I just wanted to get a run that passes.

Reply:
1.8 will be 8. The conversion is `1.x -> x` (1.9 -> 9) and `y.* -> y` (11.8 -> 11)

Reply:
Yea agreed. Just not sure when 1.8 comes up anymore. I guess not with the newer Maven compiler plugin. I'll take a look at what `TestUtil.getJavaMajorVersion()` is doing since it returns an integer anyway. So might not have to worry.

Reply:
Ah I see @asereda-gs  your comment was about `TestUtil.getJavaMajorVersion()`. Sorry :)